### PR TITLE
Swap names of WarShipDesigner and MilitaryShipDesigner

### PR DESCRIPTION
**Pull request by Morlic-fo**
As suggested by @Dilvish-fo

### CODE REVIEW

File: default/python/AI/ShipDesignAI.py [hunk 1]
```diff
@@ -1581,18 +1581,18 @@ def _combat_rating(self):
         return combat_stats.get_rating(enemy_stats=self.additional_specifications.enemy)
 
 
-class WarShipDesigner(ShipDesigner):
+class MilitaryShipDesigner(ShipDesigner):
     basename = "Warship (do not build me)"
```
Comment (Dilvish-fo) [inline default/python/AI/ShipDesignAI.py]: Seems to me we should probably also change this 'Warship' to something like 'Military ship base'

**Review by Dilvish-fo — APPROVED** [review]

File: default/python/AI/ShipDesignAI.py (hunk 1 above)
Comment (Morlic-fo) [reply default/python/AI/ShipDesignAI.py]: We have the ```TroopShipDesignerBaseClass``` and ```ColonisationShipDesignerBaseClass```

So, I would go with MilitaryShipDesignerBaseClass``` unless you have a better idea including the other base classes...

Comment (Dilvish-fo) [reply default/python/AI/ShipDesignAI.py]: Are you talking about line 1584?  For that either the current state of the PR or else MilitaryShipDesignerBaseClass would either be fine with me.  I was referring to the 'Warship' in line 1585, though (I am pretty sure that is the line I had selected when I made the comment, but I should have double checked how things appeared once I actually entered the comment)

Comment (Morlic-fo) [reply default/python/AI/ShipDesignAI.py]: Oh, you are talking about the ```basename``` variable. Good point, I overlooked that.

**Pull request merged**